core/trino-spi/src/main/java/io/trino/spi/connector/SystemTableProvider.java

```diff
@@ -11,17 +11,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.hive;
-
-import io.trino.spi.connector.ConnectorSession;
-import io.trino.spi.connector.SchemaTableName;
-import io.trino.spi.connector.SystemTable;
+package io.trino.spi.connector;
```

Comment:
What's the benefit of putting this in SPI ?
You can either put it in trino-plugin-toolkit or just make a copy in iceberg

Reply:
> or just make a copy in iceberg

Copy will not allow making `SystemTableProviders` which could be used with more than one connector.

> You can either put it in trino-plugin-toolkit

I will not argue if SPI is a better place, but I placed `SystemTableProvider` in the same place where `SystemTable` is defined.

Reply:
`SystemTableProvider` moved to `trino-plugin-toolkit`